torch_brain/transforms/unit_filter.py

```diff
@@ -0,0 +1,73 @@
+from typing import Callable
+
+import numpy as np
+
+from temporaldata import ArrayDict, Data, IrregularTimeSeries, RegularTimeSeries
+
+
+class FilterUnit:
```

Comment:
This is a class, and so should not be named with an action-phrase. `UnitFilter` seems better. (sry for being nit picky)